src/Analysis/GWASResults/Views/Home/InitialState/DefaultColumnManagement.js

```diff
@@ -0,0 +1,11 @@
+const DefaultColumnManagement = {
+  runId: true,
```

Comment:
minor comment: I think this can be improved if these variables would have names that sounded more boolean-like. E.g. `showRunId` instead of `runId`.

Reply:
That's a great idea: I have renamed the columnManagement variables: 

```
const DefaultColumnManagement = {
  showRunId: true,
  showWorkflowName: true,
  showDateSubmitted: true,
  showJobStatus: true,
  showDateFinished: true,
  showViewDetails: true,
  showActions: true,
};```